include/deal.II/grid/tria_accessor.h

```diff
@@ -1238,6 +1238,21 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
   double diameter () const;
 
   /**
+   * Return a pair of the center and the radius of the smallest enclosing
+   * sphere of the object.
+   *
+   * The function implements Ritter's O(n) algorithm to get a good approximation
+   * of the bounding ball. Starting from the good initial guess (center of the
+   * largest diagonal), the algorithm tests other vertices to be within the ball.
+   * If any such vertex is found, the new approximation of the bounding ball is
+   * constructed (by shifting its center and increasing the radius) to cover
+   * both the current ball and the given vertex, followed by restart of the test loop.
```

Comment:
now you can remove `, followed by...` as we have a single loop to extend the initial guess from the largest diagonal.